homeassistant/components/technove/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for TechnoVE."""
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import TechnoVEDataUpdateCoordinator
+
+
+class TechnoVEEntity(CoordinatorEntity[TechnoVEDataUpdateCoordinator]):
+    """Defines a base TechnoVE entity."""
+
+    _attr_has_entity_name = True
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return information about this TechnoVE station."""
+        return DeviceInfo(
+            connections={
+                (CONNECTION_NETWORK_MAC, self.coordinator.data.info.mac_address)
```

Comment:
You're using `self.coordinator.data` a lot, maybe store it in a local var to make it cleaner and a bit more performant?